mappings/net/minecraft/world/chunk/WorldChunk.mapping

```diff
@@ -60,11 +60,20 @@ CLASS net/minecraft/class_2818 net/minecraft/world/chunk/WorldChunk
 		ARG 1 levelTypeProvider
 	METHOD method_12208 createBlockEntity (Lnet/minecraft/class_2338;)Lnet/minecraft/class_2586;
 		ARG 1 pos
-	METHOD method_12210 getEntities (Ljava/lang/Class;Lnet/minecraft/class_238;Ljava/util/List;Ljava/util/function/Predicate;)V
+	METHOD method_12210 collectEntitiesByClass (Ljava/lang/Class;Lnet/minecraft/class_238;Ljava/util/List;Ljava/util/function/Predicate;)V
+		COMMENT Collects a list of entities of some runtime type and appends them to the given list. The type of
+		COMMENT the collected entities will be a subclass of the given class.
+		COMMENT
+		COMMENT @throws IllegalArgumentException if {@code entityClass} is an interface or is not a subclass
+		COMMENT of {@link Entity}.
 		ARG 1 entityClass
+			COMMENT the class object representing the type collected entities must extend
 		ARG 2 box
-		ARG 3 entityList
+			COMMENT the box in which the collected entities must be
+		ARG 3 result
+			COMMENT a list in which to store the collected entities
 		ARG 4 predicate
+			COMMENT a predicate which entities must satisfy in order to be considered
```

Comment:
```suggestion
			COMMENT a predicate which entities must satisfy in order to be collected
```
If the method is called, `collect...`, I think the comments in params should match